recipes/assimp/5.x/conanfile.py

```diff
@@ -187,45 +196,45 @@ def _patch_sources(self):
             tools.replace_in_file(os.path.join(
                 self._source_subfolder, "CMakeLists.txt"), before, after, strict=False)
         # Take care to not use these vendored libs
-        vendors = ["poly2tri", "rapidjson", "unzip", "utf8cpp", "zip"]
+        vendors = ["poly2tri", "rapidjson", "utf8cpp", "zip", "unzip", "stb", "zlib", "clipper"]
         if tools.Version(self.version) < "5.1.0":
             vendors.append("irrXML")
         else:
-            vendors.extend(["pugixml", "draco"])
+            vendors.extend(["pugixml", "draco", "openddlparser"])
```

Comment:
Why openddlparser is not unvendored in assimp < 5.1.0 also?

Reply:
Before 5.1.0, assimp used a modified and incompatible version of openddlparser